Security.mediawiki

```diff
@@ -16,9 +16,11 @@ Details on the way we classify security bugs can be [[Security Severity Ratings|
 == Security at Mozilla ==
```

Comment:
looks like the previous change on this page wasn't sync'd, if you edit from git locally try: ```git pull --rebase <origin|upstream|however your upstream is called>```